### PR TITLE
Features/Kitchen-class

### PR DESCRIPTION
**Pull request by angelewilliams**
# Description
Change Pantry to Kitchen Class. Added functionality to Kitchen to house the pantry and methods to compare recipe and create a grocery list with needed amounts of ingredients. 

Addresses # (issue)
#57 #56

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] npm test

# Checklist:
- [x] My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  I have commented my code, particularly in hard-to-understand areas
- [x]  I have made corresponding changes to the documentation
- [x]  My changes generate no new warnings
- [x]  I have added tests that prove my fix is effective or that my feature works
- [x]  New and existing unit tests pass locally with my changes
- [x]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/classes/Kitchen.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      })
+      return groceryListItem
+    })
+
+  }
+
+}
+
+
+// unique ingredients = ingredientsData
```
Comment (michaelputnam67) [inline src/classes/Kitchen.js]: Please delete this commented-out code!

File: test/Kitchen-test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    it('should be able to store userdata pantry', () => {
+        expect(kitchen.pantry).to.equal(usersData[0].pantry)
+    });
+
+    it('should instantiate kitchen in User', () => {
+        expect(kitchen).to.deep.equal(user.kitchen)
+    });
+
+    it('should be able to check if pantry has the ingredients to make recipe', () => {
+        // recipe result is the recipe ingredients without the one ingredient we have enough for
```
Comment (michaelputnam67) [inline test/Kitchen-test.js]: Also remove this code!

**Review by michaelputnam67 — COMMENTED** [review]
Looks great ! awesome job with the kitchen functionality!

File: src/classes/Kitchen.js (hunk 1 above)
Comment (angelewilliams) [reply src/classes/Kitchen.js]: Thanks for pointing this out!

**Pull request merged**